test/integration-test/commands/show/show-schema.test.ts

```diff
@@ -0,0 +1,12 @@
+import { expect, test } from '@oclif/test'
+import { publicDescription } from '../../helpers/constants'
+
+describe('User shows schema', () => {
+  test
+    .stdout()
+    .command(['show schema', 'TestSchemaPublicV1-12'])
+    .it('show a schema given an ID', (ctx) => {
+      expect(ctx.stdout).to.contains(publicDescription)
+      expect(ctx.stdout).to.contains('TestSchemaPublicV1-12')
```

Comment:
can we have this as a constant exported maybe from 
`/test/integration-test/helpers/constants.ts` ?